assets/src/edit-story/components/inspector/document/publish/publishTime.js

```diff
@@ -140,14 +141,18 @@ function PublishTime() {
         anchor={dateFieldRef}
         isOpen={showDatePicker}
         placement={'bottom-end'}
-        maxHeight={350}
       >
-        <DateTime
-          value={date}
-          onChange={handleDateChange}
-          is12Hour={use12HourFormat}
-          forwardedRef={dateTimeNode}
-        />
+        {({ propagateDimensionChange }) => (
+          <DateTime
+            value={date}
+            onChange={(value, close = false) => {
+              handleDateChange(value, close);
+              propagateDimensionChange();
```

Comment:
@miina this is currently not triggered when switching between views, which makes it require a scroll before the change is in effect. I tried using `onViewChange` from the `react-calendar` but that did not work either, any ideas?